libs/cardano-ledger-binary/cardano-ledger-binary.cabal

```
@@ -42,7 +42,8 @@ library
     default-language: Haskell2010
     ghc-options:
         -Wall -Wcompat -Wincomplete-record-updates
-        -Wincomplete-uni-patterns -Wredundant-constraints -Wunused-packages
+        -Wincomplete-uni-patterns -Wredundant-constraints
+    -- -Wunused-packages
```

Please, don't disable any warnings. They are there for a reason
```suggestion
        -Wincomplete-uni-patterns -Wredundant-constraints -Wunused-packages
```